docs/security-considerations.rst

```diff
@@ -6,22 +6,22 @@ While it is usually quite easy to build software that works as expected,
 it is much harder to check that nobody can use it in a way that was **not** anticipated.
 
 In Solidity, this is even more important because you can use smart contracts
-to handle tokens or even more valuable things. Furthermore, every execution of a smart
-contract happens in public as it is mostly open source.
+to handle tokens or, possibly, even more valuable things. Furthermore, every
+execution of a smart contract happens in public as it is mostly open environment.
```

Comment:
Hm, I think I throw together two concepts here and we should clarify that better. What about:
`... happens in public and, in addition to that, the source code is often available.`